src/TgSharp.Core/Session.cs

```diff
@@ -39,6 +40,10 @@ internal static Session TryLoadOrCreateNew (ISessionStore store, string sessionU
                     DataCenter = defaultDataCenter,
                 };
             }
+            session.Sequence = 0;
+            byte[] randomSessionId = new byte[8];
+            RandomNumberGenerator.Create().GetNonZeroBytes(randomSessionId);
+            session.Id = BitConverter.ToUInt64(randomSessionId, 0);
```

Comment:
if this commit is supposed to fix the same problem that the CI define fixed, then the removal of CI define should be in the same commit as this